semgrep/semgrep/core_runner.py

```diff
@@ -390,9 +390,8 @@ def _add_match_times(
             }
             profiling_data.set_file_times(t.target, rule_timings, t.run_time)
 
-    @staticmethod
     def get_files_for_language(
-        language: Language, rule: Rule, target_manager: TargetManager
+        self, language: Language, rule: Rule, target_manager: TargetManager
```

Comment:
I assume this can be changed back to what it was?

Reply:
Shouldn't be exposed out of this class anyway